squad-create-reproducer

```diff
@@ -97,9 +114,13 @@ def parse_args(raw_args):
 
 def run(raw_args=None):
     args = parse_args(raw_args)
+    if not args.custom_command and not args.local:
+        logger.warning(
+            "If the --custom-command flag is not provided, this script will fetch an existing TuxRun reproducer."
+        )
```

Comment:
Not sure if we need to warn about this though.

Reply:
Correct - we changed things so this isn't an issue anymore. It collects a TuxTest reproducer now.